rust/agents/relayer/src/msg/metadata/aggregation.rs

```diff
@@ -107,7 +107,14 @@ impl AggregationIsmMetadataBuilder {
 
         let metas_and_gas_count = metas_and_gas.len();
         if metas_and_gas_count < threshold {
-            info!("Could not fetch all metadata: Found {metas_and_gas_count} of the {threshold} required ISM metadata pieces");
+            info!("Could not fetch all metadata: Found {metas_and_gas_count} of the {threshold} required ISM metadata pieces for message_id {message_id}", metas_and_gas_count=metas_and_gas_count, threshold=threshold, message_id=message.id());
+            for (module_type, ism_address) in invalid_isms {
+                info!(
+                    "Invalid ISM: {module_type} at {ism_address}",
```

Comment:
Invalid? i don't think this is the only reason why estimate gas would fail

Reply:
this means missing ism as in which don't satisfy the requirements of aggregation but good point, invalid is maybe not the best term for it.